git_machete/client.py

```diff
@@ -1487,26 +1487,26 @@ def __sync_annotations_to_definition_file(self,
         for pr in prs:
             if LocalBranchShortName.of(pr.head) in self.managed_branches:
                 debug(f'{pr} corresponds to a managed branch')
-                anno: str = f'PR #{pr.number}'
+                anno: str = f'PR #{bold(str(pr.number))}'
                 if pr.user != current_user:
-                    anno += f' ({pr.user})'
+                    anno += f' ({bold(pr.user)})'
                 upstream: Optional[LocalBranchShortName] = self.up_branch.get(LocalBranchShortName.of(pr.head))
                 if pr.base != upstream:
-                    warn(f'branch `{pr.head}` has a different base in PR #{pr.number} (`{pr.base}`) '
-                         f'than in machete file (`{upstream or "<none, is a root>"}`)')
-                    anno += f" WRONG PR BASE or MACHETE PARENT? PR has '{pr.base}'"
+                    warn(f'branch {bold(pr.head)} has a different base in PR #{bold(str(pr.number))} ({bold(pr.base)}) '
+                         f'than in machete file ({bold(upstream) or "<none, is a root>"})')
```

Comment:
Whoops here the right side of `or` will never be executed.

Aaand `bold(None)` isn't what we expect either. So probably it needs to be
```suggestion
                         f'than in machete file ({bold(upstream) if upstream else "<none, is a root>"})')
```

Reply:
Oh, great catch!